diff --git a/src/database/schema.rs b/src/database/schema.rs
index 6f72ff6..46bda97 100644
--- a/src/database/schema.rs
+++ b/src/database/schema.rs
@@ -50,14 +50,17 @@ table! {
 }
 
 table! {
+    use diesel::sql_types::*;
+    use crate::database::indicator::IndiFuncMapping;
+    use crate::database::indicator::SignalClassMapping;
     indicators (indicator_id) {
         indicator_id -> Int4,
         parent_id -> Nullable<Int4>,
         child_id -> Nullable<Int4>,
         indicator_name -> Varchar,
         shift -> Int2,
-        func -> Indifunc,
-        class -> Nullable<Signalclass>,
+        func -> IndiFuncMapping,
+        class -> Nullable<SignalClassMapping>,
         filename -> Nullable<Varchar>,
         buffers -> Nullable<Array<Int2>>,
         config -> Nullable<Array<Numeric>>,
